scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -664,33 +665,27 @@ def config_logger(self):
                        )
     return logger
 
-def clean_ivt_data(self):
-    if os.path.exists(MOTR_LOG_DIR):
-        self.logger.info("Removing addb directories")
-        dnames_addb = []
-        pattern="{}/**/addb*".format(MOTR_LOG_DIR)
-        for dname in glob.glob(pattern, recursive=True):
-            dnames_addb.append(dname)
+def remove_dirs(self, log_dir, patterns):
+    if len(patterns) == 0:
+        self.logger.info(f"Removing {log_dir}")
+        execute_command(self, f"rm -rf {log_dir}")
+        return
+    for pattern in patterns:
+        temp_dir = []
+        self.logger.info(f"Removing {pattern} directories from {log_dir}")
+        search_pat = "{}/**/{}*".format(log_dir, pattern)
+        for dname in glob.glob(search_pat, recursive=True):
+            temp_dir.append(dname)
```

Comment:
what does dname contain? Is it absoluted path or relative. If relative temp_dir need to be initialized correctly.

Reply:
It is absolute path